pachr/train.R

```diff
@@ -1,18 +1,26 @@
 library(caret)
 
 # load the CSV file from the local directory
-dataset <- read.csv("/pfs/training/iris.csv", header=FALSE)
+dataset <- read.csv(path = "/pfs/training/iris.csv", header = FALSE)
 
 # set the column names in the dataset
-colnames(dataset) <- c("Sepal.Length","Sepal.Width","Petal.Length","Petal.Width","Species")
+colnames(dataset) <- c("Sepal.Length",
+					   "Sepal.Width",
+					   "Petal.Length",
+					   "Petal.Width",
+					   "Species")
 
 # Run algorithm using 10-fold cross validation
-control <- trainControl(method="cv", number=10)
+control <- trainControl(method = "cv", number = 10)
 metric <- "Accuracy"
 
 # SVM
 set.seed(7)
-fit.svm <- train(Species~., data=dataset, method="svmRadial", metric=metric, trControl=control)
+fit.svm <- train(form = Species ~ ., 
```

Comment:
just added this for consistency, but I'm not sure if `form` is correct vs `formula`. `form` is what I saw in the docs, might wanna test to make sure it still runs